triage.md

```diff
@@ -0,0 +1,40 @@
+## Table of Contents
+- [Scope](#scope)
+- [Schedule](#schedule)
+- [What Is Triaging](#what-is-triaging)
+- [How to Triage](#how-to-triage)
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) repositories. For maintainers/contributors its a guide to ensure that we address all customer issues, address feature requests in a timely manner.
+
+## Schedule
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+Meeting link: []
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
+
+## How to Triage
+The below steps are not exhaustive and can be extended. These will be performed for opensearch-benchmark as well as opensearch-benchmark-workloads repositories. In every triage, we should try to cover atleast 10 Issues combined from below categories
```

Comment:
Should we have dual links for both repositories? For example for each type of issue, we have links to both repos?

Reply:
Could you elaborate. I did not understand

Reply:
atleast -> at least

Also, embed links to the two repositories in this sentence.

Reply:
Right now, the headers -- such as Untriaged, High Priority, Breaking, etc. --  are currently only pointing to OSB repository filtered issues. I was suggesting that we should include links pointing to filtered issues for OSB workloads repository as well

Reply:
Got it, will update for OSB workloads as well